src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -472,6 +473,48 @@ void GeneralPage::initLayout(SettingsLayout &layout)
             return fuzzyToInt(args.value, 0);
         });
 
+    layout.addTitle("Streamer Mode");
+    layout.addDescription(
+        "Chatterino can automatically change behavior if it "
+        "detects that \"OBS Studio\" is running.\nSelect which "
+        "things you want to change while streaming");
+    layout.addDropdown<int>(
```

Comment:
If you use an enum maybe you should make something similar to addKeyBoardModifierSetting (a function in this file)

Reply:
Also this shouldn't be a setting you can type stuff in

Reply:
Fixed both things in 1d08fed8, huge thanks for noticing them!